main/main.cpp

```diff
@@ -1941,7 +1943,11 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	return exit_code;
 }
 
-Error Main::setup2(Thread::ID p_main_tid_override) {
+Error Main::setup2(bool p_update_main_thread) {
+	if (p_update_main_thread) {
+		Thread::make_main_thread();
```

Comment:
If this is called from a different thread than setup, doesn't this leave two threads with the id `1`?

Reply:
Yes this is called from a different thread.